service/src/main/java/com/codeforcommunity/dataaccess/AuthDatabaseOperations.java

```diff
@@ -121,13 +122,29 @@ public UsersRecord createNewUser(
     newUser.setPrivilegeLevel(PrivilegeLevel.STANDARD);
     newUser.store();
 
+    AuditRecord audit = db.newRecord(AUDIT);
+    audit.setTableName("users");
+    audit.setTransactionType("insert");
+    audit.setResult(newUser.getId().toString());
+    audit.setUserId(newUser.getId());
+    audit.setTimestamp(new Timestamp(System.currentTimeMillis()));
```

Comment:
If this will always be `NOW()` let's just keep it internal to the DB with default values.

Reply:
will make this change